HW-11/ViewController.swift

```diff
@@ -9,11 +9,329 @@ import UIKit
+
+        fourButtonsStackView.distribution = .equalSpacing
+        fourButtonsStackView.translatesAutoresizingMaskIntoConstraints = false
+
+        storyButton.widthAnchor.constraint(equalTo: fourButtonsStackView.widthAnchor, multiplier: 0.25).isActive = true
+        noteButton.widthAnchor.constraint(equalTo: fourButtonsStackView.widthAnchor, multiplier: 0.25).isActive = true
+        photoButton.widthAnchor.constraint(equalTo: fourButtonsStackView.widthAnchor, multiplier: 0.25).isActive = true
+        clipButton.widthAnchor.constraint(equalTo: fourButtonsStackView.widthAnchor, multiplier: 0.25).isActive = true
+
+        infoButtonsStackView.spacing = 15
+        let infoButtons = [cityButton, subsButton, workButton, giftButton, infoButton]
+        for button in infoButtons {
+            guard button.titleLabel?.text != nil else { continue }
+            button.imageView?.trailingAnchor.constraint(equalTo: button.titleLabel!.leadingAnchor, constant: -10).isActive = true
+        }
+    }
+
+//    MARK: - Private func
+
+    private func createHorisontalStackView() -> UIStackView {
+        let stackView = UIStackView()
+        stackView.axis = .horizontal
+        stackView.distribution = .equalSpacing
+
+        return stackView
+    }
+
+    private func createVerticalStackView() -> UIStackView {
+        let stackView = UIStackView()
+        stackView.axis = .vertical
+
+        return stackView
+    }
+
+    private let picMain: UIImageView = {
+        let image = UIImageView(image: .init(named: "Sova"))
+        image.clipsToBounds = true
+        image.layer.cornerRadius = 65
+
+        return image
+    }()
+
+    private lazy var myName: UILabel = {
+        let label = UILabel()
+        label.text = Strings.myName
+        label.textColor = .white
+        label.font = .systemFont(ofSize: 20)
+        label.textAlignment = .left
+
+        return label
+    }()
+
+    private lazy var statusButton: UIButton = {
+        let button = UIButton(type: .system)
+        button.setTitle(Strings.statusBut, for: .normal)
+        button.tintColor = Color.systemBlue
+
+        return button
+    }()
+
+    private lazy var onlineLabel: UILabel = {
+        let label = UILabel()
+        label.text = Strings.onlinelabel
+        label.textColor = Color.onlineLabel
+        label.font = .systemFont(ofSize: 12)
+        label.textAlignment = .left
+
+        return label
+    }()
+
+    private lazy var bigButton: UIButton = {
+        let button = UIButton(type: .system)
+        button.setTitle(Strings.bigButton, for: .normal)
+        button.tintColor = .white
+        button.backgroundColor = Color.bigButton
+        button.clipsToBounds = true
+        button.layer.cornerRadius = 10
+
+        return button
+    }()
+
+
+    private func createImage(name: String) -> UIImage {
+        guard let image = UIImage(named: name)
+            else {
+                return UIImage.remove }
```

Comment:
Лучше оформлять вот так код:
```suggestion
        guard let image = UIImage(named: name) else {
            return UIImage.remove
        }
```